### DISCUSSION TITLE
Please add support for DSPy

### DISCUSSION THREAD

**Discussion by RamXX** [cloudflare/workerd#1935] [Python Packages]
Kindly requesting support for [DSPy](https://github.com/stanfordnlp/dspy), an advanced AI pipeline framework to directly program LLMs instead of directly prompting them. It's suited for advanced, production-level AI workflows. Thanks!

**Comment by afeezaziz** [comment]
Hopefully when Python becomes officially supported by Cloudflare, this package can be added. This package is very important for AI agents.
